src/renderer/apps/multisend/erc20abi.json

```diff
@@ -0,0 +1,222 @@
+[
```

Comment:
There are already multiple places you can import ERC20 ABI from. 
There is one inside contractkit: `import { ABI as Erc20ABI } from '@celo/contractkit/lib/generated/IERC20'`
There is one inside lib: https://github.com/zviadm/celoterminal/blob/main/src/lib/erc20/erc20-abi.json